python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -35,13 +35,14 @@ def list_of_strings(arg):
                     help='tensor parallel degree')
 parser.add_argument('--max_length', type=int, default=None,
                     help='max sequence length for the tokenizer')
-parser.add_argument('--do_lower_case', type=bool, default=True,
-                    help='do lower case for the tokenizer')
-parser.add_argument('--add_special_tokens', type=bool, default=True,
-                    help='the sequences will be encoded with the special tokens relative to their model')
+parser.add_argument('--disable_lower_case', action='store_true',
+                    help='do not use lower case for the tokenizer')
+parser.add_argument('--disable_special_tokens', action='store_true',
+                    help='the sequences will not be encoded with the special tokens relative to their model')
 parser.add_argument('--tensor_input_names', type=list_of_strings, default=None,
                     help='the tensor input names passed to the model')
 parser.add_argument('--task', required=False, help="The ML task name")
+parser.add_argument('--disable_vllm', action='store_true', help="Do not use vllm as the default runtime")
```

Comment:
Missing default here

Reply:
This is on/off flag argument not an option that takes value.

Reply:
Why the default is true? I think we should default to false unless user wants to explicitly turn it off and let it fallback to huggingface API.

Reply:
`action='store_true'` provide capability to set bool value if the flag is set. In this case if `--disable_vllm` is set, the value of `arg.disable_vllm` will be true and in all other cases it is false

https://docs.python.org/3/library/argparse.html#action

The experience is if `--disable_vllm` is set only then vLLM will not be used explicitly. if this flag is not set then vLLM to be used, with  huggingface API as fallback